onnx/backend/test/case/node/averagepool.py

```diff
@@ -471,12 +471,24 @@ def export_averagepool_2d_dilations() -> None:
             "AveragePool",
             inputs=["x"],
             outputs=["y"],
-            kernel_shape=[3, 3],
+            kernel_shape=[2, 2],
             strides=[1, 1],
             dilations=[2, 2],
             ceil_mode=True,
         )
-        x = (np.arange(16) + 1).astype(np.float32).reshape((1, 1, 4, 4))
```

Comment:
Minor point: Retaining a comment like `# input shape: [1, 1, 4, 4]` will be helpful.